src/main/java/frc/robot/Constants.java

```diff
@@ -174,68 +177,109 @@ public static final class AutoConstants {
-    public static final double kMaxOutput = 1.0;
+    public static final int kIntakeCurrentLimit = 70; // amps
+	  public static final double kIntakeSpeed = 1.0;
+    public static final double kOutakeSpeed = -1.0;
+    public static final double kConeIdleSpeed = 0.05;
+    public static final double kCubeIdleSpeed = -0.05;
   }
    public static final class ArmConstants {
     // Arm limits
-    public final static float REVERSE_LIMIT = 0.0f;
-    public final static float FORWARD_LIMIT = 43.5f;
-    public final static double REVERSE_SPEED = 0.5;
-    public final static double FORWARD_SPEED = 0.5;
-
-    // doubles for arm positions
-    public final static double INTAKE_ARM_GROUND = 0.0;
-    public final static double INTAKE_ARM_SHELF = 0.0;
-    public final static double SCORING_ARM_LOW = 0.0;
-    public final static double SCORING_ARM_MID = 0.0;
-    public static final Double INTAKE_ARM_STOWED = 0.0;
+    public final static float ARM_REVERSE_LIMIT = 34.0f;
+    public final static float ARM_FORWARD_LIMIT = 0.0f;
+    public final static double ARM_REVERSE_SPEED = 0.5;
+    public final static double ARM_FORWARD_SPEED = 0.5;
+
+    // Elbow limits // TODO: tune these values 
+    public final static float ELBOW_REVERSE_LIMIT = 0.0f;
+    public final static float ELBOW_FORWARD_LIMIT = 0.0f;
+    public final static double ELBOW_REVERSE_SPEED = 0.5;
+    public final static double ELBOW_FORWARD_SPEED = 0.5;
 
     /* Spark max constants */ 
     // CAN ID
     public final static int LEFT_MOTOR_CAN_ID = 9;
     public final static int RIGHT_MOTOR_CAN_ID = 10;
-    public static final boolean RIGHT_MOTOR_INVERTED = false;
-    public static final boolean LEFT_MOTOR_INVERTED = true; 
-    public static final double kArmEncoderPositionFactor = 0.0;
-    public static final double kArmEncoderVelocityFactor = 0.0;
-    public static final IdleMode kArmMotorIdleMode = IdleMode.kBrake;
+    public final static int ELBOW_MOTOR_CAN_ID = 11;
+
+    // Motor directions 
+    public static final boolean RIGHT_ARMMOTOR_INVERTED = false;
+    public static final boolean LEFT_ARMMOTOR_INVERTED = true; 
+    public static final boolean ELBOW_MOTOR_INVERTED = true;
+
+    // Absolute encoder conversions 
+    public static final double kArmEncoderPositionFactor = 2 * Math.PI;
+    public static final double kArmEncoderVelocityFactor = (2 * Math.PI) / 60.0;
+
+    public static final double kElbowEncoderPositionFactor = 2 * Math.PI;
+    public static final double kElbowEncoderVelocityFactor = (2 * Math.PI) / 60.0;
+
+    // Idle mode
+    public static final IdleMode kMotorIdleMode = IdleMode.kBrake; 
+
+    // Current limits 
     public static final int kArmMotorCurrentLimit = 40;
-    public static final int kMovingArmMotorCurrentLimit = 60;  
+    public static final int kMovingArmMotorCurrentLimit = 60;
+    public static final int kElbowMotorCurrentLimit = 60;
 
     // PID constants using custom PID gains class //TODO: tune PID values
```

Comment:
Are these PID constants tuned?  Can we delete the TODO?

Reply:
They need to be retuned with the feedforward controller